src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/Example2.java

```diff
@@ -1,20 +1,19 @@
 /*xml
 <module name="Checker">
   <module name="TreeWalker">
-    <module name="DesignForExtension">
-      <property name="ignoredAnnotations" value="Override"/>
-    </module>
+    <module name="DesignForExtension"/>
```

Comment:
please restore property or explain why it is removed.

Reply:
Done